tests/mail/tests.py

```diff
@@ -205,6 +205,22 @@ def test_cc_in_headers_only(self):
         ).message()
         self.assertEqual(message["Cc"], "foo@example.com")
 
+    def test_cc_not_repeated(self):
+        message = EmailMessage(
+            "Subject",
+            "Content",
+            "bounce@example.com",
+            ["to@example.com"],
+            cc=["foo@example.com"],
+            headers={"Cc": "override@example.com"},
+        ).message()
+
+        count_cc = 0
+        for el in list(message.raw_items()):
```

Comment:
`raw_items()` is an undocumented method. What do you think about using `get_all()` instead (e.g. `get_all("cc")`)?